userchrome-loader/loader.css

```diff
@@ -0,0 +1,17 @@
+/*  This file is intended to make it easier to load customizations from other users. It acts strictly as a loader.
+
+    Do not add any custom CSS here.
+
+    If you want to add custom CSS, create a new .CSS file under: "chrome/userchrome-loader/(name-of-new-file.css)"
+
+    You can orginize custom CSS files in subfolders as well: "chrome/userchrome-loader/new-folder/(name-of-new-file.css)"
+
+    Add an @import rule for each new file you create in this file. */
+    
+/* --- Examples --- */
+/* @import "userchrome-loader/example.css"; */
+/* @import "userchrome-loader/new-folder/example2.css"; */
+
+/*--------------------------------------------------------------------*/
+
+/* === Load custom files === */
```

Comment:
Can you make some example files in userchrome-loader/modules (make a new folder) so we can add import statements for those here? Would rather not have the loader example be 100% comments